app/test/e2e/utils/test-server.js

```diff
@@ -2,21 +2,14 @@ const chai = require('chai');
 const chaiHttp = require('chai-http');
 const nock = require('nock');
 
-let requester;
```

Comment:
please revert these changes, or every time you run a test, a new web server is started, which is unnecessary overhead.